Projects/Team3/Mosiychuk Tania/ParallelSeleniumTest/LoginPage.cs

```diff
@@ -0,0 +1,30 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+
+namespace ParallelSeleniunTest
+{
+    public class LoginPage
+    {
+        private IWebDriver driver;
+
+        public static void GoToSite(IWebDriver driver, string site)
+        {
+            driver.Navigate().GoToUrl(site);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(1000);
+        }
+        public static void Login(IWebDriver driver)
+        {
+            driver.FindElement(By.XPath("/html/body/div[1]/div[2]/div[1]/div[1]/form/div[1]/input")).SendKeys("tania2002mosiy@gmail.com");
+            driver.FindElement(By.XPath("/html/body/div[1]/div[2]/div[1]/div[1]/form/div[2]/input")).SendKeys("sm6ozi802" + Keys.Enter);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(1000);
+        }
+        
+        public static void ChekText(IWebDriver driver)
```

Comment:
it is better to declare the driver using the LoginPage constructor.
So it will give as a possibility to call driver as a class field and remove it from each method parameters